addons/account/tests/common.py

```diff
@@ -162,18 +162,12 @@ def setUpClass(cls):
 
         # ==== Payment methods ====
         bank_journal = cls.company_data['default_journal_bank']
-        in_outstanding_account = cls.env['account.account'].create({
-            'name': "Outstanding Receipts",
-            'code': 'OSTR00',
-            'reconcile': True,
-            'account_type': 'asset_current'
-        })
-        out_outstanding_account = cls.env['account.account'].create({
-            'name': "Outstanding Payments",
-            'code': 'OSTP00',
-            'reconcile': True,
-            'account_type': 'asset_current'
-        })
+        in_outstanding_account = cls.env['account.chart.template'].ref('account_journal_payment_debit_account_id')
+        in_outstanding_account.code = 'OSTR00'
```

Comment:
I don't get this change actually, you shouldn't have that xmlid when testing on runbot :thinking: 
But using `raise_if_not_found` wouldn't work either because you would then set `.code` on `False`